src/components/settings/SettingsHeightmapTab.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <div>
+        <v-card flat>
+            <v-card-text>
+                <div class="d-flex align-center">
+                    <v-icon style="opacity: 0.7">{{ mdiGrid }}</v-icon>
+                    <v-card-title class="mx-n2">
+                        {{ $t('Settings.HeightmapTab.Navigation') }}
+                    </v-card-title>
+                    <v-divider class="ml-3"></v-divider>
+                </div>
+                <settings-row :title="$t('Settings.HeightmapTab.ColorSchemes')">
+                    <v-select
+                        v-model="colorScheme"
+                        :items="availableColorSchemes"
+                        hide-details
+                        outlined
+                        dense
+                        attach></v-select>
+                </settings-row>
+            </v-card-text>
+        </v-card>
+    </div>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component'
+import { Mixins, Watch } from 'vue-property-decorator'
+import BaseMixin from '@/components/mixins/base'
+import SettingsRow from '@/components/settings/SettingsRow.vue'
+import Panel from '@/components/ui/Panel.vue'
+import { mdiGrid } from '@mdi/js'
+
+@Component({
+    components: {
+        Panel,
+        SettingsRow,
+    },
+})
+export default class SettingsHeightmapTab extends Mixins(BaseMixin) {
+    mdiGrid = mdiGrid
+
+    availableLanguages: { text: string; value: string }[] = []
+
+    async created() {
```

Comment:
i think you don't need this function, because this is only for general > languages